dist/lovefield.js

```diff
@@ -1,6 +1,6 @@
-if(!self.window){window=this;}
+if(!self.window){window=self;}
 (function(){'use strict';var $jscomp = {scope:{}}, goog = goog || {};
-goog.global = this;
+goog.global = self;
```

Comment:
checking... it looks like the change to goog.global was not needed. I switched it back to 'this' with no problems. window=self is the only change needed for the fix. without it we are getting:

Uncaught (in promise) Error: lf is not defined(…)